windows/utils/win_pause_resume_win_update.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022-2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Disable Windows auto update and pause Windows Update 7 days,
+# or delete this pauing Windows Update config in registry to
+# resume Windows Update.
+# Paramters:
+#   win_update_ops: valid values are 'pause' or 'resume'.
+#
+- name: "Check required parameter"
+  ansible.builtin.assert:
+    that:
+      - win_update_ops is defined
+      - win_update_ops in ['pause', 'resume']
+    fail_msg: >-
+      Parameter 'win_update_ops': {{ win_update_ops | default('') }} is not set correctly,
+      valid value is 'pause' or 'resume'.
+
+- name: "Set fact of PowerShell script file name"
+  ansible.builtin.set_fact:
+    win_update_script: "{{ win_update_ops }}_windows_update.ps1"
+
+# Transfer PowerShell script to guest OS
+- name: "Set fact of PowerShell script file path"
+  ansible.builtin.set_fact:
+    local_ps_file: "scripts/{{ win_update_script }}"
+    remote_ps_file: "C:\\{{ win_update_script }}"
+- name: "Copy file from local to guest OS"
+  include_tasks: win_copy_file_from_local.yml
+  vars:
+    src_path_local: "{{ local_ps_file }}"
+    dest_path_remote: "C:\\"
+- name: "Check if file {{ remote_ps_file }} exists"
+  include_tasks: win_check_file_exist.yml
+  vars:
+    win_check_file_exist_file: "{{ remote_ps_file }}"
```

Comment:
What if remote_ps_file doesn't exist?

Reply:
Add a check for `win_check_file_exist_result`, failed if it's false.